external/fv3fit/fv3fit/keras/_models/shared/loss.py

```diff
@@ -0,0 +1,86 @@
+import dataclasses
+import numpy as np
+import tensorflow as tf
+
+
+def _standard_scaled_mse(std):
+    std = tf.constant(std, dtype=std.dtype)
+
+    def custom_loss(y_true, y_pred):
+        return tf.math.reduce_sum(
+            tf.math.reduce_mean(tf.math.square((y_pred - y_true) / std), axis=0)
+        )
+
+    return custom_loss
+
+
+def _standard_scaled_mae(std):
+    std = tf.constant(std, dtype=std.dtype)
+
+    def custom_loss(y_true, y_pred):
```

Comment:
Nit- standard and uniform_standard functions handle the custom loss differently- can we just handle them both the same way for consistency?

Reply:
What do you mean when you say they "handle the custom loss differently"? The configuration is meant to change the behavior of the loss function so that it uses individual std values or the same std value for all levels/features.